### PR TITLE
DEVX-2072: Registry handlers

### PR DESCRIPTION
**Pull request by lenaorobei**
## Description

- changed CLI to use `path` to the template's code instead of `packageUrl`
- added checks for `install.yml` file
- changed `lint` configuration by adding a required semicolon
- removed `checkNodeVersion.js` since we check Node version during the installation
- added tests
- updated docs
- removed unnecessary tests

## How Has This Been Tested?
```
npm test
```

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: src/checks/containsKeyword.js [hunk 1]
```diff
@@ -10,39 +10,38 @@ OF ANY KIND, either express or implied. See the License for the specific languag
...
-                message: '',
-                status: 'pass',
-            }
-        } else {
-            return {
-                message:
-                    'ecosystem:aio-app-builder-template must be one of the keywords in the NPM package',
-                status: 'fail',
-            }
+    const index = fileData.keywords.findIndex((element) => {
```
Comment (slitviachenko) [inline src/checks/containsKeyword.js:26]: we can use a shorter form
```
element => element.includes('aio-app-builder-template')
```

File: src/checks/descriptionNotEmpty.js [hunk 2]
```diff
@@ -10,32 +10,26 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-async function descriptionNotEmpty(metadataJSON) {
-    if (!metadataJSON) {
+/**
+ * Check that package.json contains a description
+ *
+ * @param fileData
```
Comment (slitviachenko) [inline src/checks/descriptionNotEmpty.js:16]: @param {object} fileData

File: src/commands/check.js [hunk 3]
```diff
@@ -10,49 +10,58 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-const { checkTemplateMetadata } = require('../lib/templateMetadata')
+const { checkTemplateMetadata } = require('../lib/templateMetadata');
 
+/**
+ * Outputs check results
+ *
+ * @param results
```
Comment (slitviachenko) [inline src/commands/check.js:18]: @param {object} results

File: src/commands/check.js [hunk 4]
```diff
@@ -10,49 +10,58 @@ OF ANY KIND, either express or implied. See the License for the specific languag
...
+ * @param outputInJson
```
Comment (slitviachenko) [inline src/commands/check.js:19]: @param {boolean} outputInJson

File: src/commands/check.js [hunk 5]
```diff
@@ -10,49 +10,58 @@ OF ANY KIND, either express or implied. See the License for the specific languag
...
-    // parse package URL input for package name
-    let p_url = new URL(packageUrl)
-    let packageName = p_url.pathname.split('package/')[1]
-    let results = await checkTemplateMetadata(packageName)
-    process.exitCode = results.stats.failures !== 0 ? 1 : 0
-    outputResults(results, outputInJson)
+/**
+ * Run checks on template package
+ *
+ * @param path
```
Comment (slitviachenko) [inline src/commands/check.js:56]: @param {string} path

File: src/commands/check.js [hunk 6]
```diff
@@ -10,49 +10,58 @@ OF ANY KIND, either express or implied. See the License for the specific languag
...
+ * @param options
```
Comment (slitviachenko) [inline src/commands/check.js:57]: @param {object} options

File: src/lib/templateMetadata.js [hunk 7]
```diff
@@ -10,31 +10,35 @@ OF ANY KIND, either express or implied. See the License for the specific languag
...
-    }
-}
-
-// run checks on NPM package
-async function checkTemplateMetadata(packageName) {
-    let templateMetadata = await getTemplateMetadata(packageName)
+/**
+ * Run checks on template package
+ *
+ * @param path
```
Comment (slitviachenko) [inline src/lib/templateMetadata.js:38]: @param {string} path

**Review by AmyJZhao — APPROVED** [review]

**Review by slitviachenko — APPROVED** [review]
Everything looks good to me. I left non-critical comments.

**Pull request merged**